frontend/src/utils/constants.js

```diff
@@ -23,6 +23,7 @@ export const ApiRoutes = Object.freeze({
   ORGANIZATIONS: baseRoot + '/organizations',
   FACILITIES: baseRoot + '/facilities',
   ORGANIZATIONS_FACILITIES: baseRoot + '/organizations/:organizationId/facilities',
+  ORGANIZATIONS_USERS: baseRoot + '/organizations/:organizationId/users',
```

Comment:
We haven't been too consistent here but it would make sense to reuse the ORGANIZATIONS constant rather than repeating baseRoot + '/organizations'.

Reply:
Fixed.